### ISSUE TITLE
Некорректный font-family в header и cards, неверный цвет у кнопки в первом promo

### ISSUE THREAD

**Issue by alexandrYun** [isakura313/landing_ozon#6]
В header у .logo_second и в courses у .course_card__link не задан шрифт Ralewey
У кнопки в первом promo задан некорректный цвет --main-bg-color вместо --second-bg-color

**Comment by isakura313** [comment]
Так, это во второй странице макета у кнопки в первом лого этот цвет, а не в первом, который мы делаем. А вот в лого и вправду не хватает шрифта, серьезная ошибка. Хороший issue!
